xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-filters/xwiki-platform-notifications-filters-api/src/main/java/org/xwiki/notifications/filters/expression/EventProperty.java

```diff
@@ -90,5 +90,11 @@
     /**
      * The version of the page concerned by the event.
      */
-    DOCUMENT_VERSION
+    DOCUMENT_VERSION,
+    /**
+     * The unique identifier of the instance in the cluster.
+     * 
+     * @since 14.7RC1
```

Comment:
Don't you miss the unstable too? Not sure if it makes sense for a new enum value.

Reply:
Yes and also in Event#getRemoteObservationId() it seems.